src/components/calcite-picker-row/calcite-picker-row.scss

```diff
@@ -0,0 +1,36 @@
+:host {
+  display: flex;
+  align-items: center;
+  padding: 12px;
+  background-color: white;
+  box-shadow: 0px 2px 3px 0px rgba(0, 0, 0, 0.75);
+  margin-bottom: 12px;
+  cursor: pointer;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host(.highlight) {
+  border-left: 3px solid transparent;
```

Comment:
I think we need RTL styles for this

Reply:
@asangma Would we also swap the positioning of the icon and secondary icon?

Reply:
Shouldn't flex handle that?

Reply:
yeah it should